spark/v3.1/spark/src/main/java/com/netease/arctic/spark/ArcticSparkCatalog.java

```diff
@@ -206,15 +201,37 @@ public Table createTable(
     }
   }
 
-  private Schema checkAndConvertSchema(Schema schema, Map<String, String> properties) {
+  private Schema checkAndConvertSchema(StructType schema, Map<String, String> properties) {
+    Schema convertSchema;
+    try {
+      CatalogMeta catalogMeta = client.getCatalog(catalog.name());
+      String catalogType = catalogMeta.getCatalogType();
+      if (CATALOG_TYPE_HIVE.equals(catalogType)) {
```

Comment:
In my opinion, the hive table use timestamp without zone only, and the iceberg table's timestamp type is dependent on the configuration which is the default with zone.